package.json

```
@@ -3,8 +3,8 @@
   "private": true,
   "version": "0.0.1",
   "scripts": {
-    "audit": "npm audit && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit'",
-    "audit:fix": "npm audit fix && lerna exec --concurrency 1 'echo $LERNA_PACKAGE_NAME && npm audit fix'",
+    "audit": "npm audit & lerna exec 'echo $LERNA_PACKAGE_NAME & npm audit'",
```

```suggestion
    "audit": "npm audit --production & lerna exec --no-bail 'echo $LERNA_PACKAGE_NAME & npm audit --production'",
```
So, `--no-bail` makes sure every package gets checked.  `--production` is, unsurprisingly, to ignore dev dependencies.

---

I think we should not ignore dev dependencies in the commands. We should ignore them after the audit. The reason being technically, in the client every dependency should actually be a dev dependency.

---

I guess I'm missing something, then. I thought that the packages that get bundled up and become the site are the normal dependencies, with the dev dependencies just being used for testing, watching and so on.

Anyway, we want `--no-bail`, regardless.

---

Well, sort of. Anything that has to be bundled and sent in the browser, the likes of test runner, sass, etc. is packaged using webpack.

Anything that is gatsby is used to generate the static pages. So a lot is actually dev deps.

I would add the `--no-bail` per your recommendation